google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingImpl.java

```diff
@@ -781,7 +785,12 @@ static ListLogEntriesRequest listLogEntriesRequest(
     String filter = FILTER.get(options);
     if (filter != null) {
       builder.setFilter(filter);
+    } else {
```

Comment:
I think in Python and Go, we went with checking whether filter contained "timestamp" string. Rather than whether filter exists.

Reply:
Hm.. OK. Let me add that